Makefile

```diff
@@ -11,6 +11,8 @@ KG_IMAGE_NAME:=jupyter-incubator/kernel-gateway-extras
 KG_CONTAINER_NAME:=kernel-gateway
 HTTP_PORT?=3000
 HTTPS_PORT?=3001
+TEST_CONTAINER_NAME:=$(DASHBOARD_CONTAINER_NAME)-test
+TEST_IMAGE_NAME:=$(TEST_CONTAINER_NAME)
```

Comment:
For consistency, it should be named with a `jupyter-incubator` namespace like the other images:

`TEST_IMAGE_NAME:=jupyter-incubator//$(TEST_CONTAINER_NAME)`